pkg/collector/python/version_test.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package python
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/util/cache"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetPythonVersion(t *testing.T) {
+	cache.Cache.Set(pythonInfoCacheKey, "3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0]", cache.NoExpiration)
+	require.Equal(t, "3.10.6", GetPythonVersion())
+}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Any particular reason for not testing the "n/a" case like before ?